tests/input/examples/personinfo_basic/data/Agent-expected-001.yaml

```diff
@@ -0,0 +1,17 @@
+id: P:001
```

Comment:
set this up before seeing your PR with the new naming conventions/file structure